docs/SUMMARY.md

```diff
@@ -135,3 +135,11 @@
         * [Using CircleCI](guides/ci/circleci.md)
     * [Static Linking](guides/static_linking.md)
 * [Platform Support](platform_support.md)
+* [Tutorials](tutorials/README.md)
+    * [Basics](tutorials/01_basics/README.md)
+        * [Hello World](tutorials/01_basics/01_hello_world.md)
+        * [Variables](tutorials/01_basics/02_variables.md)
+        * [Math](tutorials/01_basics/03_math.md)
```

Comment:
This clearly expresses the order of lessons. I like to see that order while editing to know which contents have already been established in previous lessons.
After the content is published I don't expect to insert new lessons in the middle because every lesson builts on the previous ones (to some degree) and new contents should be added to the end. In case there is a need to insert a new lesson somewhere in between, it could just get an existing number, maybe with a order suffix.

Reply:
We can use 10 / 20 / 30 numbers to insert things in the middle if we need. I usually do that. Leftovers of GW Basic 🙈

Reply:
That sounds like a good idea. We should avoid having more than 10 lessons in a course, so that fits.

Reply:
And I suppose the courses don't need to be ordered, so we can remove the prefix from `01_basics`.